.github/ISSUE_TEMPLATE/add_new_activity.yaml

```
@@ -28,6 +28,7 @@ body:
         - Workshop
         - Media
         - Evento
+        - Escola
```

```suggestion
        - Educação
```

Foi discutido em equipa e fica melhor Educação, pelo menos como versão inicial :)